src/docs/apiary.apib

```diff
@@ -73,6 +93,16 @@ Returns all currently configured flows
                         "label": "Tempalte"
                     }
                 ],
+                "flowRemoteNodes": [
+                    {
+                        "id": "kelvin",
+                        "image": "raulnegreiros/kelvin:3.0.0-alpha2"
```

Comment:
hum... it is not nice to put personal things on dojot's repository. It should be nice to change it to dojot

Reply:
Hi @raulnegreiros, thanks for your feedback. I have sent de corrections.